src/config/dynamic-theme-fixes.config

```diff
@@ -12015,11 +12015,15 @@ thunderbird.net
 
 INVERT
 .w-48
+a[title="Thunderbird"] > svg
 
 CSS
 body {
     background-image: none !important;
 }
+body {
```

Comment:
Don't fiddle around with the background-colors of html/body. The background-color inconsistency is currently by-design.

Reply:
> The background-color inconsistency is currently by-design.

Could I ask why? I have now removed it.